presto-main/src/main/java/com/facebook/presto/operator/WindowInfo.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.operator.window.WindowPartition;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import javax.annotation.concurrent.Immutable;
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.collect.Iterables.concat;
+
+public class WindowInfo
+        implements Mergeable<WindowInfo>, OperatorInfo
+{
+    private final List<DriverWindowInfo> windowInfos;
```

Comment:
Info was too hard to send, which I described here: https://github.com/prestodb/presto/pull/7661#issuecomment-307498062 but it was due to the idea of displaying whole distribution instead of std dev only.
In order to display the distribution you need to carry the information over (it's not mergeable). In the same time, std dev can be pre-accumulated and delivered to coordinator for final display. Each driver have only one `DriverWindowInfo` in the `windowInfos` list. Although, I use a `List` to make it easily mergeable on the coordinator (coordinator can simply call `OperatorInfo.mergeWith()` as it does for any other mergeable operator).